### PR TITLE
Feature/inset map overlay

### PR DESCRIPTION
**Pull request by CloudNiner**
Companion PR to https://github.com/cartodb-org/cartodb.js/pull/5

There's a ton of edge cases here for inset map positioning, if you see any bugs write em up and I'll try to take a look.

Fixing up tests now.

bbg-public-inset-map

### CODE REVIEW

File: lib/assets/javascripts/cdb [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit 3415ddd07ee8b6c04237025cc8a1d164192ae5a8
+Subproject commit b308a025d124af7b11144a95adf9bca8e47b7176
```
Comment (flibbertigibbet) [inline lib/assets/javascripts/cdb]: I suppose this will need to be updated again once the companion PR goes in.

**Comment by flibbertigibbet** [comment]
Tried it out and it seems to be behaving well.

+1
